aiconfig-docs/wip_docs/readme_v0.md

```diff
@@ -0,0 +1,568 @@
+<div align="center"><picture>
```

Comment:
Hmmmm I meant more like moving the specific subsections that were deleted into their proper subpages in the wip_docs or the main docs. For example now there's no place where the API is documented...